### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SmartSergiZ**
- [DEMO LINK](https://SmartSergiZ.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://SmartSergiZ.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,67 @@
...
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  margin: 0;
+  box-sizing: border-box;
```
Comment (alexander-ignatow) [inline src/style.css:11]: Setting `box-sizing` individually for each class is not a great idea - as there are few scenarios when one needs to vary it.

In most cases, including yours, it is better to set a single box-sizing value for all elements and vary it only when needed

Try to fix this issue - it looks minor in this task but may become painful in large projects.

Here is a good article on box-sizing: https://css-tricks.com/box-sizing/

And here is a good way to set box-sizing: https://css-tricks.com/box-sizing/#aa-good-better-and-probably-best-box-sizing-reset-methods

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+  color: #00acdc;
+}
+
+.logo__img {
+  display: flex;
+  align-items: center;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```
Comment (alexander-ignatow) [inline src/style.css:55]: It is better to set colors via CSS variables - to be able to control them.  This even more important if the color is reused.
Check this: https://www.w3schools.com/css/css3_variables.asp

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+}
+
+.nav__link.is-active::after {
+  display: block;
+  height: 4px;
+  width: 100%;
+  background-color: #00acdc;
+  position: absolute;
+  bottom: 0;
+  border-radius: 8px;
```
Comment (alexander-ignatow) [inline src/style.css:65]: The border-radius is bigger than the height - an obvious typo here :)
It does not have any effect in this case, still please fix it.

**Review by alexander-ignatow — CHANGES_REQUESTED** [review]
Good solution! Visually everything is ok, but let's improve the code quality a bit.

**Comment by SmartSergiZ** [comment]
tnx)
all changes have already been added

**Review by roman-mirzoian — APPROVED** [review]
Well done 👍

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,70 @@
...
+  --text-color: #000;
+  --link-color: #00acdc;
+}
+
+nav,
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
```
Comment (alexander-ignatow) [inline src/style.css:11]: box-sizing is not inherited - you should set it globally via `*`

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,70 @@
...
+}
+
+.header {
+  margin: 0;
```
Comment (alexander-ignatow) [inline src/style.css:15]: No margins on BEM blocks

**Review by alexander-ignatow — CHANGES_REQUESTED** [review]
Very good - some things are fixed, still some remain

Also pleas make sure to deploy your latest solution ☝️

**Review by BudnikOleksii — APPROVED** [review]
Good job👍

**Pull request closed without merging**